src/face_registry.hh

```diff
@@ -20,6 +20,9 @@ public:
 
     CandidateList complete_alias_name(StringView prefix,
                                       ByteCount cursor_pos) const;
+
+    auto aliases() { return m_aliases; }
```

Comment:
Using `auto` here means we will return a copy of the aliases HashMap every time we call this function, you could use `auto&` here, but for methods like that I'd rather have the type spelled out, `const AliasMap&`. Also, make the method itself const (put const between `()` and `{`).